doc/examples/template.json

```diff
@@ -7,17 +7,23 @@
 	 * to set the timer's period to 0
 	 */
 	"tasks" : {
+		/* no 'loop' value at task level means loop indefinitely (-1) */
 		"thread0" : {
-			"instance" : 1,
-			"loop" : -1,
-			"run" :   10000,
-			"sleep" : 0,
+			"instance" : 1,  /* number of threads to be created */
+			"loop" : -1,  /* loop forever (see 'duration' below) */
+
+			"run" :   10000,  /* create load [ns]: when awaken,
```

Comment:
run it's in [us] (as stated in tutorial.txt)

Reply:
sorry, missed that one.